debian/control

```diff
@@ -0,0 +1,33 @@
+Source: txfixtures
+Maintainer: Debian Python Modules Team <python-modules-team@lists.alioth.debian.org>
+Uploaders: Free Ekanayaka <freee@debian.org>
```

Comment:
Free has no more interest in this package - this might be a historic thing.

Reply:
Added LP dev email here.